---
title: "f0171_v"
date: 2020-08-09T01:36:29+88:00
draft: false
weight: 11710

---

### Log reference: <no value>

```
    loading [Config]:  ./tests/functests/upconfig.yml
    Main config:
                 Version -> 1.0.0
                  RefDir -> ./tests/functests
                 WorkDir -> cwd
              AbsWorkDir -> /up_project/up
                TaskFile -> f0171
                 Verbose -> v
              ModuleName -> self
               ShellType -> /bin/sh
           MaxCallLayers -> 8
                 Timeout -> 3600000
     MaxModuelCallLayers -> 256
    work dir: /up_project/up
    -exec task: task
    loading [Task]:  ./tests/functests/f0171
    module: [self], instance id: [dev], exec profile: []
    Task1: [task ==> task:  ]
    -Step1: [: task fails
     ]
    cmd( 1):
    open a file .....
     .. ok
    cmd( 2):
          exec wait -> exit status 1
          exit status 1
     .. failed(suppressed if it is not the last step)
    Step Finally:
    (*utils.ExecResult)({
      Cmd: "echo \"reading the file\" |grep \"cause an exception\"",
      Code: 1,
      Output: "",
      ErrMsg: "exit status 1"
    })
    
    <nil>
    
    =Task2: [task ==> close_file:  ]
    --Step1: [close_file: ensure the opened file is closed
     ]
    cmd( 1):
    close the file .....
     .. ok
    . ok
     WARN: [No rescued in task level] - [please assess the panic problem and cause, fix it before re-run the task]
    task finally ->   ERROR: Failed And Not Ignored! [You may want to continue and ignore the error]
    
    
```

##### Logs with different verbose level
* [f0171 log - verbose level v](../../logs/f0171_v)
* [f0171 log - verbose level vv](../../logs/f0171_vv)
* [f0171 log - verbose level vvv](../../logs/f0171_vvv)
* [f0171 log - verbose level vvvv](../../logs/f0171_vvvv)
* [f0171 log - verbose level vvvvv](../../logs/f0171_vvvvv)

##### References
* [Related Chapter](../../flow-controll/f0171)
